package.json

```diff
@@ -263,7 +263,8 @@
     "grunt/minimatch": "^3.1.2",
     "**/globule/minimatch": "^3.1.2",
     "ansi-regex": "^5.0.1",
-    "testcafe/**/jsonwebtoken": "^9.0.0"
+    "testcafe/**/jsonwebtoken": "^9.0.0",
+    "stylis": "4.3.0"
```

Comment:
not `^4.3.0` intentionally?

Reply:
Yeah, seems like `stylis` can have breaking changes in behavior some of the exported utility functions between minor and potentially patch versions